index.html

```diff
@@ -268,8 +269,16 @@ <h2 class="work-title">My Recent Works</h2>
     <!-- about section -->
     <section class="about-section">
       <div class="about-text">
-        <img src="icons/desktopimage/illustration-about me-1-desktop@2x.png" alt="" class="image1">
-        <img src="icons/desktopimage/illustration-about-me-2-desktop@2x.png" alt="" class="image2">
+        <img
+          src="icons/desktopimage/illustration-about me-1-desktop@2x.png"
+          alt=""
+          class="image1"
```

Comment:
- [ ] You didn't give any alt in the image. Kindly give alts in all the images in your project. Alts are the words which appear if your image will not show up. It is helpfull for blind users because they will hover on these images and alts will give the information that what this image is about such as **project**, **right-icon** etc.